### ISSUE TITLE
Remove opal-browser dependency

### ISSUE THREAD

**Issue by wied03** [zetachang/react.rb#133]
In order to keep the core GEM light, might want to see if this can be removed and if necessary, put it on a higher level GEM.

**Comment by ajjahn** [comment]
@wied03 I agree with this. Opal browser is only gaining the ability to [attach components to browser Elements](https://github.com/zetachang/react.rb/blob/master/lib/react/top_level.rb#L89-L95). Really, we could continue to support that, but remove the requirement. Users can add depend on opal-browser specifically if they want that behavior.

**Comment by sollycatprint** [comment]
This issue was moved to reactrb/reactrb#133
